fiftyone/server/lightning.py

```
@@ -126,7 +128,10 @@ class StringLightningResult(LightningResult):
 async def lightning_resolver(
     input: LightningInput, info: Info
 ) -> t.List[LightningResults]:
-    dataset: fo.Dataset = fo.load_dataset(input.dataset)
+    run = lambda: get_view(input.dataset, reload=True)
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `run` as a `def`

(E731)